mp/src/game/client/c_baseanimating.cpp

```diff
@@ -3610,6 +3610,10 @@ void C_BaseAnimating::DoAnimationEvents( CStudioHdr *pStudioHdr )
 //-----------------------------------------------------------------------------
 bool C_BaseAnimating::DispatchMuzzleEffect( const char *options, bool isFirstPerson )
 {
+	static ConVarRef r_drawviewmodel("r_drawviewmodel");
```

Comment:
Instead of having a static `ConVarRef` you can extern this cvar as it is included. Eg. `extern ConVar r_drawviewmodel`.